tests/e2e/config/screenshots.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * E2E setup for screenshots.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { clearScreenshots } from '../utils';
+
+clearScreenshots();
```

Comment:
A bit out of scope here but a bit of a pain point I ran into when repeatedly re-running E2E tests.